WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductFromImage/AddProductFromImageScannedTextView.swift

```diff
@@ -0,0 +1,40 @@
+import SwiftUI
+
+/// Selectable and editable scanned text for generating product details in the "add product from image" flow in `AddProductFromImageView`.
+struct AddProductFromImageScannedTextView: View {
+    @ObservedObject private var viewModel: AddProductFromImageViewModel.ScannedTextViewModel
+
+    init(viewModel: AddProductFromImageViewModel.ScannedTextViewModel) {
+        self.viewModel = viewModel
+    }
+
+    var body: some View {
+        HStack {
+            TextField("", text: $viewModel.text)
+                .font(.body)
+            Spacer()
+            Button(action: {
+                viewModel.isSelected.toggle()
+            }) {
+                Image(systemName: viewModel.isSelected ? "checkmark.circle.fill" : "circle")
+            }
+        }
+        .padding(.vertical, insets: Layout.verticalPadding)
```

Comment:
Nit: this can be simplified as `.padding(.vertical, 6)`. I think insets should only be needed if we have special padding for each edge.

Reply:
Updated in https://github.com/woocommerce/woocommerce-ios/pull/10220/commits/d34d4aaef801c5cd23b3e2f53e048fc9bed70245